stable/mlflow/templates/serviceaccount.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ include "mlflow.serviceAccountName" . }}
+  name: {{ include "mlflow.fullname" . }}
```

Comment:
I guess simply using the mlflow.fullname template should be enough or? Or do we explicitly want to have the ability to have a serviceAccountName different from fullname?

Reply:
I think `mlflow.fullname` is enough for now for this chart